agent/lib/kontena/service_pods/common.rb

```diff
@@ -1,9 +1,36 @@
 require_relative '../helpers/event_log_helper'
+require_relative '../helpers/weave_helper'
 
 module Kontena
   module ServicePods
     module Common
       include Kontena::Helpers::EventLogHelper
+      include Kontena::Helpers::WeaveHelper
+
+      # Docker create configuration for ServicePod
+      # @param [ServicePod] service_pod
+      # @return [Hash] Docker create API JSON object
+      def config_container(service_pod)
```

Comment:
I think this Common could be converted to a class such as `Kontena::ServicePods::DockerContainer`